extensions/wikia/Flags/Flags.i18n.php

```diff
@@ -28,7 +28,7 @@
 	'flags-special-create-form-save' => 'Save',
 	'flags-special-create-form-save-success' => 'The flag has been added!',
 	'flags-special-create-form-save-failure' => 'Unfortunately, an error happened. Can you try again?',
-	'flags-special-autoload-delete-confirm' => 'Deleting this flag will also remove it from all articles. Are you sure?',
+	'flags-special-autoload-delete-confirm' => 'Deleting the $1 flag will also remove it from all articles. Are you sure?',
```

Comment:
np, hope we won't loose track of it. Maybe a subtask with a checklist of places that needs docs to lookup before dev merge?